CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
```

Comment:
@v-dobrev, I still think something is missing in this sentence 😁

Reply:
This file was added by @adrienbernede in the branch `v1.0` and from there it made its way into `v2.0` -- I assume this is some form of WIP.